wasmplugin/plugin.go

```diff
@@ -41,6 +41,8 @@ type corazaPlugin struct {
 
 	waf coraza.WAF
 
+	metricsLabels []map[string]string
```

Comment:
I wonder why we just don't store a map. Are you concerned about duplicates? I think on duplicated we just go with the first one and ignore the rest as only one value is going to be matched anyways?